### PR TITLE
Update dsh-bio modules to version 2.0.4

### PR DESCRIPTION
**Pull request by heuermh**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `<SOFTWARE>.version.txt` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: software/dshbio/filtergff3/main.nf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    } else {
+        container "quay.io/biocontainers/dsh-bio:2.0.4--hdfd78af_0"
+    }
+
+    input:
+    tuple val(meta), path(gff3)
+
+    output:
+    tuple val(meta), path("*.gff3.gz"), emit: gff3
+    path "*.version.txt"             , emit: version
```
Comment (drpatelh) [inline software/dshbio/filtergff3/main.nf:26]:
```suggestion
    path "*.version.txt"              , emit: version
```

File: software/dshbio/splitgff3/main.nf [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    } else {
+        container "quay.io/biocontainers/dsh-bio:2.0.4--hdfd78af_0"
+    }
+
+    input:
+    tuple val(meta), path(gff3)
+
+    output:
+    tuple val(meta), path("*.gff3.gz"), emit: gff3
+    path "*.version.txt"             , emit: version
```
Comment (drpatelh) [inline software/dshbio/splitgff3/main.nf:26]:
```suggestion
    path "*.version.txt"              , emit: version
```

**Review by drpatelh — APPROVED** [review]
Looks 👌🏽 @heuermh ! Thanks!

**Comment by heuermh** [comment]
Thank you, @drpatelh!

**Pull request merged**